includes/class-evf-emails.php

```diff
@@ -456,7 +456,7 @@ public function everest_forms_html_field_value( $html = true ) {
 				}
 
 				// If there's the export data filter, utilize that and re-loop promptly.
-				if ( has_filter( "everest_forms_field_exporter_{$field['type']}" ) && 'signature' !== $field['type'] ) {
+				if ( has_filter( "everest_forms_field_exporter_{$field['type']}" ) || 'signature' === $field['type'] ) {
```

Comment:
Since we have added 'signature as case to field_exporter, i think we dont need to add additional || (OR)  'signature' === $field['type']